checkout/access-checkout-android/src/main/java/com/worldpay/access/checkout/model/CardDefaults.kt

```diff
@@ -9,8 +9,8 @@ package com.worldpay.access.checkout.model
  * @property year a default year validation rule
  */
 data class CardDefaults(
-	val pan: CardValidationRule?,
-	val cvv: CardValidationRule?,
-	val month: CardValidationRule?,
-	val year: CardValidationRule?
+	val pan: CardValidationRule? = null,
```

Comment:
Could we set the defaults in here instead of the parser? Or is there a scenario that it could be a different set of defaults.

Reply:
Currently we allow merchants to set their own defaults if they wish